composer.json

```diff
@@ -65,6 +65,7 @@
         "phpunit/phpunit": "^9.5",
         "psalm/plugin-phpunit": "^0.16",
         "psalm/plugin-symfony": "^3.0",
+        "rector/rector": "dev-main",
```

Comment:
I think rector just released a patch version. Can you see if the fix is included?